couchable/__init__.py

```diff
@@ -41,10 +41,10 @@
 --README.txt--
 """
 
-from core import CouchableDb
-from core import registerDocType, CouchableDoc
-from core import registerAttachmentType, CouchableAttachment
-from core import registerPickleType, registerNoneType, registerUncouchableType
-from core import custom_packer
-from core import doGzip, doGunzip
-from core import newid
+from .core import CouchableDb
+from .core import registerDocType, CouchableDoc
+from .core import registerAttachmentType, CouchableAttachment
+from .core import registerPickleType, registerNoneType, registerUncouchableType
+from .core import custom_packer
+from .core import doGzip, doGunzip
+from .core import newid
```

Comment:
I'm not sure why this was working in the first place, but to get it to run locally I needed to make a relative import statement.